client/app/routes/resources/resources.scss

```diff
@@ -0,0 +1,32 @@
+.resources-image {
+  position: absolute;
+    width: 300px;
+  	margin-top: $base-padding;
+    margin-right: $base-padding;
+    margin-left: $base-padding;
+}
+
+.vision-board {
+    top: 80px;
+    left: 355px; 
+}
+.goal-setting {
```

Comment:
It would be more elegant/readable/maintainable if all of these settings could shared a common class.